src/java/picard/analysis/CollectGcBiasMetrics.java

```diff
@@ -82,13 +71,6 @@
     @Option(shortName = StandardOptionDefinitions.REFERENCE_SHORT_NAME, doc = "The reference sequence fasta file.")
```

Comment:
I think you need to remove the reference as it is already a standard argument of command_line program. I'm not sure why it was an explicit field before....perhaps someone else can explain.